miniapp/src/main/java/com/rakuten/mobile/miniapp/RealMiniApp.kt

```diff
@@ -0,0 +1,37 @@
+package com.rakuten.mobile.miniapp
+
+import com.rakuten.mobile.miniapp.display.Displayer
+import com.rakuten.mobile.miniapp.miniapp.MiniAppDownloader
+import com.rakuten.mobile.miniapp.miniapp.MiniAppLister
+
+@Suppress("TodoComment", "NotImplementedDeclaration")
+internal class RealMiniApp(
+    val qrCodeParser: QrCodeParser,
+    val miniAppDownloader: MiniAppDownloader,
+    val displayer: Displayer,
+    val miniAppLister: MiniAppLister
+) : MiniApp {
+
+    override suspend fun listMiniApp(
+        success: (List<MiniAppInfo>) -> Unit,
+        error: (Exception) -> Unit
+    ) {
+        TODO("not implemented")
+    }
+
+    override suspend fun create(
+        appId: String,
+        success: (MiniAppView) -> Unit,
+        error: (Exception) -> Unit
+    ) {
+        TODO("not implemented")
+    }
+
+    override suspend fun createFromQrCode(
```

Comment:
Similar comment on this one: maybe we shouldn't put this in yet because we still won't be implementing this for a while.

- [ ] Remove `createFromQrCode` api